src/api/tree/mutate/deleteNode.ts

```diff
@@ -12,6 +12,27 @@ import { useApiKey } from "~/util/store/apiKey";
 import { useNamespace } from "~/util/store/namespace";
 import { useToast } from "~/design/Toast";
 
+const updateCache = (
```

Comment:
I creared a `updateCache` method here as well. This one has a second parameter, which is a bit more trickey to type manually.